infra/base-images/base-builder/compile_centipede

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo -n "Skipping compilation; using precompiled centipede"
+
+export BIN_DIR="$SRC/centipede/bazel-bin/"
+
+cp "$BIN_DIR/centipede" "$OUT"
+cp "$BIN_DIR/libcentipede_runner.pic.a" "$LIB_FUZZING_ENGINE"
+mkdir "$OUT/corpus"
+mkdir "$OUT/workdir"
```

Comment:
Why is this needed? Can this be done in the dockerfile? I don't think this should be done here (probably)

Reply:
`"$OUT/corpus"` stores the seed inputs and generated inputs. I reckon there must be a more general place, as all fuzzers need it, but I was not able to find it (Dockerfile did not mention "$OUT/" at all).

"$OUT/workdir" might not have a better place than this, as it is specific to `Centipede`. `Centipede` uses this dir to store a file that encapsulates the whole corpus for internally usage.

Reply:
Will these be generated by Centipede automatically if they don't exist? Do we need to be the ones to create them?

Reply:
Pretty sure `Centipede` needs `workdir`, I will double-check `corpus`.
@jonathanmetzman taught me that these dirs should not be created at here, I will remove these lines later.